google-cloud-storage/lib/google/cloud/storage/policy.rb

```diff
@@ -106,6 +107,29 @@ def initialize etag, version
       #   end
       #
       class PolicyV1 < Policy
+        ##
+        # Returns the version 1 bindings as a hash that associates roles with an
+        # array of members. See [Understanding
+        # Roles](https://cloud.google.com/iam/docs/understanding-roles) for a
+        # listing of primitive and curated roles. See [Buckets:
+        # setIamPolicy](https://cloud.google.com/storage/docs/json_api/v1/buckets/setIamPolicy)
+        # for a listing of values and patterns for members.
+        #
+        # @raise [RuntimeError] If version is greater than 1.
```

Comment:
I copied this from somewhere, and didn't look at it closely. This might need to be changed.

Reply:
I moved this to a YARD `@attr` tag up above on line 89.